### PR TITLE
ADEN-5100 Add ads to featured video

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/ArticleVideo/scripts/ooyala-player.js [hunk 1]
```diff
@@ -111,14 +111,27 @@ define('ooyala-player', function () {
 		$('.oo-state-screen-info').css('display', '');
 	};
 
-	OoyalaHTML5Player.initHTML5Player = function (videoElementId, playerParams, videoId, onCreate, autoplay) {
+	OoyalaHTML5Player.initHTML5Player = function (videoElementId, playerParams, videoId, onCreate, autoplay, vastUrl) {
```
Comment (WikiaTech) [inline extensions/wikia/ArticleVideo/scripts/ooyala-player.js:114]: MAJOR Function has 6 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

File: extensions/wikia/ArticleVideo/scripts/article-featured-video.js [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+require([
+	'wikia.window',
+	'wikia.onScroll',
+	'wikia.tracker',
+	'ooyala-player',
+	'wikia.abTest',
+	'wikia.articleVideo.videoFeedbackBox',
+	require.optional('ext.wikia.adEngine.adContext'),
+	require.optional('ext.wikia.adEngine.video.vastUrlBuilder')
+], function (window, onScroll, tracker, OoyalaPlayer, abTest, VideoFeedbackBox, adContext, vastUrlBuilder) {
```
Comment (WikiaTech) [inline extensions/wikia/ArticleVideo/scripts/article-featured-video.js:10]: MAJOR Function has 8 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 264 issues
* ![MAJOR][MAJOR] 167 major
* ![MINOR][MINOR] 94 minor
* ![INFO][INFO] 3 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [ooyala-player.js#L99](https://github.com/Wikia/app/blob/11c95687c1e5996b6d03b712dd73453422a07ee3/extensions/wikia/ArticleVideo/scripts/ooyala-player.js#L99): Either make this concatenation explicit or cast one operand to a number. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS3402)
1. ![MAJOR][MAJOR] [google_ima.js](https://github.com/Wikia/app/blob/11c95687c1e5996b6d03b712dd73453422a07ee3/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js): File "google_ima.js" has 5319 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS104)
1. ![MAJOR][MAJOR] [google_ima.js#L1](https://github.com/Wikia/app/blob/11c95687c1e5996b6d03b712dd73453422a07ee3/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js#L1): Make this function anonymous by removing its name: 'function() {...}'. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ANamedFunctionExpression)
1. ![MAJOR][MAJOR] [google_ima.js#L1](https://github.com/Wikia/app/blob/11c95687c1e5996b6d03b712dd73453422a07ee3/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js#L1): Remove the unused function parameter "require". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
1. ![MAJOR][MAJOR] [google_ima.js#L1](https://github.com/Wikia/app/blob/11c95687c1e5996b6d03b712dd73453422a07ee3/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js#L1): Remove the unused function parameter "module". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
1. ![MAJOR][MAJOR] [google_ima.js#L1](https://github.com/Wikia/app/blob/11c95687c1e5996b6d03b712dd73453422a07ee3/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js#L1): Remove the unused function parameter "exports". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
1. ![MAJOR][MAJOR] [google_ima.js#L4](https://github.com/Wikia/app/blob/11c95687c1e5996b6d03b712dd73453422a07ee3/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js#L4): Add the "let", "const" or "var" keyword to this declaration of "OO" to make it explicit. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2703)
1. ![MAJOR][MAJOR] [google_ima.js#L7](https://github.com/Wikia/app/blob/11c95687c1e5996b6d03b712dd73453422a07ee3/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js#L7): Remove the unused function parameter "module". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
1. ![MAJOR][MAJOR] [google_ima.js#L7](https://github.com/Wikia/app/blob/11c95687c1e5996b6d03b712dd73453422a07ee3/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js#L7): Remove the unused function parameter "exports". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
1. ![MAJOR][MAJOR] [google_ima.js#L20](https://github.com/Wikia/app/blob/11c95687c1e5996b6d03b712dd73453422a07ee3/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js#L20): Remove the unused function parameter "require". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'
[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

File: extensions/wikia/ArticleVideo/scripts/article-featured-video.js (hunk 2 above)
Comment (dianafa) [inline extensions/wikia/ArticleVideo/scripts/article-featured-video.js:10]: what about changing `window` to `win`? :)

**Comment by dianafa** [comment]
LGTM

**Comment by WikiaTech** [comment]
SonarQube analysis reported 264 issues
* ![MAJOR][MAJOR] 167 major
* ![MINOR][MINOR] 94 minor
* ![INFO][INFO] 3 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [ooyala-player.js#L99](https://github.com/Wikia/app/blob/7c968b3f84795bf7fa735a425fb05e99df402f19/extensions/wikia/ArticleVideo/scripts/ooyala-player.js#L99): Either make this concatenation explicit or cast one operand to a number. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS3402)
1. ![MAJOR][MAJOR] [google_ima.js](https://github.com/Wikia/app/blob/7c968b3f84795bf7fa735a425fb05e99df402f19/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js): File "google_ima.js" has 5319 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS104)
1. ![MAJOR][MAJOR] [google_ima.js#L1](https://github.com/Wikia/app/blob/7c968b3f84795bf7fa735a425fb05e99df402f19/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js#L1): Make this function anonymous by removing its name: 'function() {...}'. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ANamedFunctionExpression)
1. ![MAJOR][MAJOR] [google_ima.js#L1](https://github.com/Wikia/app/blob/7c968b3f84795bf7fa735a425fb05e99df402f19/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js#L1): Remove the unused function parameter "require". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
1. ![MAJOR][MAJOR] [google_ima.js#L1](https://github.com/Wikia/app/blob/7c968b3f84795bf7fa735a425fb05e99df402f19/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js#L1): Remove the unused function parameter "module". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
1. ![MAJOR][MAJOR] [google_ima.js#L1](https://github.com/Wikia/app/blob/7c968b3f84795bf7fa735a425fb05e99df402f19/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js#L1): Remove the unused function parameter "exports". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
1. ![MAJOR][MAJOR] [google_ima.js#L4](https://github.com/Wikia/app/blob/7c968b3f84795bf7fa735a425fb05e99df402f19/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js#L4): Add the "let", "const" or "var" keyword to this declaration of "OO" to make it explicit. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2703)
1. ![MAJOR][MAJOR] [google_ima.js#L7](https://github.com/Wikia/app/blob/7c968b3f84795bf7fa735a425fb05e99df402f19/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js#L7): Remove the unused function parameter "module". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
1. ![MAJOR][MAJOR] [google_ima.js#L7](https://github.com/Wikia/app/blob/7c968b3f84795bf7fa735a425fb05e99df402f19/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js#L7): Remove the unused function parameter "exports". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
1. ![MAJOR][MAJOR] [google_ima.js#L20](https://github.com/Wikia/app/blob/7c968b3f84795bf7fa735a425fb05e99df402f19/extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js#L20): Remove the unused function parameter "require". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'
[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Pull request merged**